### PR TITLE
fix: mac VO announce same phrase & timeout config

### LINKED ISSUE #25: Unable to announce same message more than once
Reproducible on [demo page](https://elements.polymer-project.org/elements/iron-a11y-announcer?view=demo:demo/index.html). 
Clicking any single announce button multiple times in a row will only announce the message once.
Tested on Chrome 53 and Canary.

### PR DESCRIPTION
**Pull request by e111077**
fixes #25

### CODE REVIEW

**Comment by e111077** [comment]
also changed head to body because mac VO will not announce aria-live when it is in head

**Comment by e111077** [comment]
tests pass locally. I think travis is just dead, Jim

File: iron-a11y-announcer.js [hunk 1]
```diff
@@ -107,5 +113,5 @@ IronA11yAnnouncer.requestAvailability = function() {
     IronA11yAnnouncer.instance = document.createElement('iron-a11y-announcer');
   }
 
-  document.head.appendChild(IronA11yAnnouncer.instance);
+  document.body.appendChild(IronA11yAnnouncer.instance);
```
Comment (rictic) [inline iron-a11y-announcer.js]: What happens if this is called before document.body exists? I seem to recall we've had at least one bug like that before

Comment (e111077) [reply iron-a11y-announcer.js]: TIL. That's a good point. I should wait for document load

File: iron-a11y-announcer.js [hunk 2]
```diff
@@ -107,5 +113,12 @@ IronA11yAnnouncer.requestAvailability = function() {
     IronA11yAnnouncer.instance = document.createElement('iron-a11y-announcer');
   }
 
-  document.head.appendChild(IronA11yAnnouncer.instance);
+  if (document.body) {
+    document.body.appendChild(IronA11yAnnouncer.instance);
+  } else {
+    document.addEventListener('load', function() {
+      console.log('load');
```
Comment (rictic) [inline iron-a11y-announcer.js]: console.log

**Review by rictic — APPROVED** [review]

File: iron-a11y-announcer.js (hunk 2 above)
Comment (e111077) [reply iron-a11y-announcer.js]: omg imma dummy

**Pull request merged**